packages/lodestar-beacon-state-transition/src/fast/block/isValidIndexedAttestation.ts

```diff
@@ -40,9 +40,12 @@ export function isValidIndexedAttestation(
   const pubkeys = indices.map((i) => epochCtx.index2pubkey[i]);
   const domain = getDomain(config, state, DomainType.BEACON_ATTESTER, indexedAttestation.data.target.epoch);
   const signingRoot = computeSigningRoot(config, config.types.AttestationData, indexedAttestation.data, domain);
-  return verifyAggregate(
-    pubkeys,
-    signingRoot,
-    indexedAttestation.signature.valueOf() as Uint8Array,
+  return pubkeys.reduce(
+    (aggregate, pubkey) => {
+      return aggregate.add(pubkey);
+    }, pubkeys.shift()
+  ).verifyMessage(
```

Comment:
What would happen here if pubkeys is an empty array?

Reply:
It would fail much earlier 😄 there is a check for that